src/common/Header/HeaderNavigationButton.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import type { Navigation } from '../../types';
+
+import * as React from 'react';
+import InjectedComponent from '../InjectedComponent';
+import { withNavigation } from 'react-navigation';
+import { COLORS } from '../../theme';
+import IconButton from '../buttons/IconButton';
+
+type InjectedProps = {|
+  navigation: Navigation,
+|};
+
+type Props = {
+  to: string,
```

Comment:
nit: make this a bit more descriptive. `toView` maybe?  Try to use some common language that they use with react-navigation.

Reply:
ok, changed to `toRoute`